adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -225,6 +259,10 @@ private void performAcquireTokenRequest(final CallbackHandler callbackHandle,
         //       broker 2) broker doesn't return any token back.
         final AuthenticationResult authenticationResultFromSilentRequest = tryAcquireTokenSilent(authenticationRequest);
         if (isAccessTokenReturned(authenticationResultFromSilentRequest)) {
+            mAPIEvent.setWasApiCallSuccessful(true, null);
+            mAPIEvent.setCorrelationId(authenticationRequest.getCorrelationId().toString());
```

Comment:
>mAPIEvent.setCorrelationId(authenticationRequest.getCorrelationId().toString()); [](start = 12, length = 80)

i'm lost here, why do we have the correlationid set everywhere? cann't we set it per request?